jipdate.py

```diff
@@ -227,7 +232,7 @@ def parse_status_file(jira, filename):
     # [SWG-28]
     # [LITE-32]
     # etc ...
-    regex = r"^\[([A-Z]+-\d+)\]\n$"
+    regex = r"^\[(.*)\]\n$"
```

Comment:
If one could query Jira for a list of "key types" [SWG, VIRT ... etc], then one could just do that and then later compare. I did some quick checks but couldn't find any way to do it (which isn't the same as it isn't possible).

Seems like we would like to use this in different ways (surprise, surprise!). For me it's sufficient to just write top as a general email where I can list other non-Jira things too and then down to the end I have all the Jira issues, one by one.

I can see the benefit of writing like @chazy is doing right above. But doing like so will make parsing much harder. I believe to figure out the start/end for that will still require some other marker.